terraform-module-standards.md

```diff
@@ -345,14 +345,19 @@ You should make updating the version pin a regular practice:
 ```hcl
 provider "equinix" {
   source = "equinix/equinix"
-  version = "= 1.11.0-alpha.2"
+  version = "= 1.11.0"
```

Comment:
This change is sufficient compared to the version that was previously defined. To @ocobleseqx's point, we could use `>=` rather than `=`.

I'll approve as-is, we can fix that separately or while this is still open. I'll leave it to @ocobleseqx.

Reply:
We have documentation in the next paragraph that suggests exactly what we should do here:

> For example, using `~>1.11` indicates the module will work with any provider version `>= 1.11, < 2.0`.